src/components/ChartContainer.js

```diff
@@ -63,16 +60,44 @@ const ChartContainer = forwardRef(
     const container = useRef();
     const chart = useRef();
     const downloadButton = useRef();
+    const transformComponentRef = useRef(null);
```

Comment:
Not familiar with useRef. Why is this one null as opposed to other 3?

Reply:
I mainly wanted to set the default to `null` instead of how the above are defaulted to `undefined`. You can read more about `useRef` [here](https://reactjs.org/docs/hooks-reference.html#useref)

In this case I'm using it to access the context from the zoom library